services/thumbnail_queue/thumbnail_queue.py

```diff
@@ -0,0 +1,137 @@
+from threading import Thread
+import time
+
+import tornado.ioloop
+import tornado.web
+import asyncio
+import tornado.escape
+import json
+
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker, scoped_session
+
+from baselayer.app.models import init_db
+from baselayer.app.env import load_env
+from baselayer.log import make_log
+from baselayer.app.flow import Flow
+
+from skyportal.models import (
+    DBSession,
+    Obj,
+)
+
+env, cfg = load_env()
+log = make_log('thumbnail_queue')
+
+init_db(**cfg['database'])
+
+Session = scoped_session(sessionmaker())
+
+queue = []
```

Comment:
After looking at the code, since the ZTF cutouts are ingested with an endpoint and not this queue, I feel more confident that this should work just fine 👍

Reply:
yes but for now we just totally fail so slow should improve what we do now.